kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/pedestals/pedestal_crushing.js

```diff
@@ -46,13 +46,8 @@ events.listen('recipes', (event) => {
     data.recipes.forEach((recipe) => {
         const re = event.custom({
             type: 'pedestals:pedestal_crushing',
-            ingredient: {
-                item: recipe.input
-            },
-            result: {
-                item: recipe.output,
-                count: recipe.count
-            }
+            ingredient: Ingredient.of(recipe.input),
+            result: Item.of(recipe.output, recipe.count)

```

Comment:
Did you test this in-game?

Reply:
Yes, it doesn't affect any of the recipes in the script (apart from fixing the flour recipe)

Reply:
Huh, I thought `.toJson()` was needed, but.. cool!